.github/workflows/main.yaml

```diff
@@ -14,7 +14,7 @@ env:
   extensions: "json"
   cache-version: "1"
   composer-version: "v2"
-  composer-install: "composer update --no-interaction --no-progress --no-suggest --prefer-dist --prefer-stable"
```

Comment:
Just for info: why are we removing this?

Reply:
because this 
![obrazek](https://user-images.githubusercontent.com/6066243/148390783-1e791965-036f-484a-9c3c-024548c1b796.png)

Reply:
Nice, thanks!